src/components/CodeReminderModal/index.jsx

```diff
@@ -131,13 +132,25 @@ export class BaseCodeReminderModal extends React.Component {
 
     if (isBulkRemind && !data.selectedCodes.length) {
       options.code_filter = selectedToggle;
-    } else if (isBulkRemind && data.selectedCodes.length) {
-      options.assignments = data.selectedCodes.map(code => ({
-        email: code.assigned_to,
```

Comment:
we have to keep this `email` argument.
As your PR will merge before, and it will brake the system if we will not pass this argument.

Reply:
@HammadAhmadWaqas Please review the changes. I have addressed the feedback. Also, can you please verify these changes by locally checking out this branch?